client-java/sql/src/main/java/org/evomaster/client/java/sql/heuristic/ColumnReference.java

```diff
@@ -0,0 +1,44 @@
+package org.evomaster.client.java.sql.heuristic;
+
+import java.util.Objects;
+
+public class ColumnReference {
```

Comment:
provide JavaDoc to explain what this class is for

Reply:
added to this class and other classes similar to this one.